internal/pkg/warewulfd/warewulfd.go

```
@@ -21,30 +20,28 @@ func RunServer() error {
 		for range c {
 			err := LoadNodeDB()
 			if err != nil {
-				wwlog.Printf(wwlog.WARN, "Could not load database: %s\n", err)
+				fmt.Printf("ERROR: Could not load database: %s\n", err)
 			}
 		}
 	}()
 
 	err := LoadNodeDB()
 	if err != nil {
-		wwlog.Printf(wwlog.WARN, "Could not load database: %s\n", err)
+		fmt.Printf("ERROR: Could not load database: %s\n", err)
```

Why not return the error here?